staging/src/k8s.io/apiserver/pkg/util/flowcontrol/gen_test.go

```diff
@@ -562,6 +562,7 @@ func genResourceRule(rng *rand.Rand, pfx string, mayMatchClusterScope, matchAllR
 		rr.ClusterScope = true
 		rr.Namespaces = []string{fcv1a1.NamespaceEvery}
 	}
+	rr.ResourceNames = []string{fcv1a1.ResourceNameEvery}
```

Comment:
TODO, OK if it is in a follow-up PR: exercise the other possibilities here.

Reply:
fixed